### PR TITLE
Adds Storybook support

### PR DESCRIPTION
**Pull request by aendrew**
This adds Storybook and loads in VVC/g-components. It also:

- [x] Removes logic around using npm link — don't link anymore! Build new components in `app/components`, if they're useful to copy into `g-components` or VVC, do that after publishing!
- [x] Removes superfluous and deprecated papaparse dependency
- [x] Updates Mocha
- [x] Updates g-components to latest

### CODE REVIEW

File: app/util/isVVCInstalled.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+module.exports.getVVCRoot = () => {
+  let vvcRoot = false;
+
+  try {
+    vvcRoot = resolve(dirname(require.resolve('@financial-times/vvc/package.json')), 'src');
+  } catch (e) {
+    if (!runOnce) {
+      console.info(
+        'VVC not installed. To install, ensure you have your GitHub SSH key added to the SSH agent, then npm install'
```
Comment (joannaskao) [inline app/util/isVVCInstalled.js:13]: Might as well add a link to the Gitihub documentation so people know how to do that
```suggestion
        'VVC not installed. To install, ensure you have your GitHub SSH key added to the SSH agent (https://docs.github.com/en/enterprise/2.17/user/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent), then npm install'
```

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```
Comment (joannaskao) [inline .storybook/preview.js:3]: This is to trigger actions like `onClick`? When is this used? I'm sure this makes sense, but I can't think of where this gets used.

**Review by joannaskao — CHANGES_REQUESTED** [review]
This is a great idea, though can you also update the README so that people know how to run storybook with a starter kit project? Does any component in the components folder show up in storybook? Didn't automatically show up for me when I just tried it. 

Also, is there a way to get the word "Core" out of these? 
Screen Shot 2020-09-09 at 11 41 39

And is there a way to label these as VVC components in just starter-kit or would that require changing VVC (which I don't think we should do)? 
Screen Shot 2020-09-09 at 11 42 45

File: .storybook/preview.js (hunk 2 above)
Comment (aendrew) [reply .storybook/preview.js:3]: I have no idea tbth; the scaffolder added it. I think it's to create actions from event handlers? Not entirely sure.

**Review by tlfrd — COMMENTED** [review]
I'm having issues running starter-kit. I'm getting
```Error [ERR_INVALID_PACKAGE_TARGET]: Invalid "exports" target "./src" defined for './' in the package config /Users/XXX/Code/starter-kit-test/node_modules/@financial-times/g-components/package.json```

And the warning
```WARN unable to find package.json for @financial-times/g-components```
when running storybook (which does work for me)

File: README.md [hunk 3]
```diff
@@ -20,31 +20,35 @@ What the setup script does:
...
-This assumes your project keeps most of your data in the `client/` and `config/` folders.
+
+This assumes your project keeps most of your data in the `app/` and `config/` folders.
 
 #### Using the built in tasks
 
 You can run the following tasks from within your project directory:
 
 - `npm start` — starts up a development server and opens your app in a web browser. The dev server will automatically reload your browser when files change.
+- `npm run storybook` — loads up [StorybookJS](https://storybook.js.org/) as a development environment for building components.
```
Comment (joannaskao) [inline README.md:51]:
```suggestion
- `npm run storybook` — loads up [StorybookJS](https://storybook.js.org/) as a development environment for building components. Stories created in the `app/components` folder will automatically show up in the storybook.
```

**Review by joannaskao — APPROVED** [review]
Ran it locally and it works great!

**Pull request merged**